packages/@aws-cdk/aws-codestarnotifications/package.json

```diff
@@ -90,8 +91,14 @@
   "engines": {
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
-  "stability": "experimental",
-  "maturity": "cfn-only",
+  "stability": "stable",
+  "maturity": "stable",
+  "awslint": {
+    "exclude": [
+      "no-unused-type:@aws-cdk/aws-codestarnotifications.NotifyOnEventOptions",
+      "props-physical-name:@aws-cdk/aws-codestarnotifications.RuleProps"
```

Comment:
Both of these should be removed.

Reply:
Already removed.

Reply:
Currently, It still has `props-physical-name:@aws-cdk/aws-codestarnotifications.RuleProps` awslint error, can I add this to resolve this error?
```
yarn run v1.22.10
$ cdk-build
error: [awslint:props-physical-name:@aws-cdk/aws-codestarnotifications.RuleProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name
```

Reply:
Try calling it `notificationRuleName` instead, and see if it passes then.

BTW, I think we might consider renaming the types here to `(I)NotificationRule`, to differentiate them from `IRule` in CloudWatch.

Reply:
No problem, I will adjust.

Reply:
This name will be more consistent between CDK modules!

Reply:
Do all of `IRuleTarget` and `IRuleSource` or `*Rule*` have to be rename?

Reply:
Yeah, let's call them `IRuleNotificationsTarget` and `IRuleNotificationsSource`.

Reply:
Done, I change to `(I)NotificationRule*`.